config/dev/hmc_values.yaml

```diff
@@ -1,5 +1,5 @@
 image:
-  repository: hmc/controller
+  repository: localhost/hmc/controller
```

Comment:
Why this is needed?

Reply:
```
Digest: sha256:367e2184d70c7bdda48e8a444ea1bf21f03587ee0975ed96f6d2739c15007ddd
/Users/bnr/work/github/hmc/bin/kind-v0.23.0 load docker-image hmc/controller:latest -n hmc-dev
enabling experimental podman provider
ERROR: image: "hmc/controller:latest" not present locally
make: *** [dev-push] Error 1
```

The image that is created is actually tagged as

```
podman images
REPOSITORY              	TAG        	IMAGE ID  	CREATED         	SIZE
localhost/hmc/controller	latest     	b62d60df23f2  About a minute ago  61.8 MB
```

Reply:
btw, tested this on both docker and podman.

Reply:
Ok, got it, it's needed for podman to function